database/migrations/2022_09_16_081901_create_sites_table.php

```diff
@@ -21,6 +21,7 @@ public function up()
             $table->string('public')->nullable();
             $table->boolean('track_clicks')->default(true);
             $table->boolean('track_movements')->default(true);
+            $table->boolean('active')->default(true);
```

Comment:
Please create a new migration to add this column, existing applications won't run this piece of code and miss out on this column, or force them to refresh their database and loose data.

Reply:
resolved